src/item_factory.cpp

```diff
@@ -1004,6 +1004,26 @@ void Item_factory::check_definitions() const
         std::string msg;
         const itype *type = &elem.second;
 
+        if( !type->item_tags.count( "TARDIS" ) ) {
+            bool is_container = false;
+            for( const pocket_data &pocket : type->pockets ) {
+                if( pocket.type == item_pocket::pocket_type::CONTAINER ) {
+                    is_container = true;
+                    // no need to look further
+                    break;
+                }
+            }
```

Comment:
Could write this with `std::any_of` and a lambda.  Personal taste as to whether that's preferable.